sensor_msgs/NavSatFix goal
int8 type  # 0: post without visual marker, 1: post with visual marker, 2: gate with visual markers